synapse/lib/stormtypes.py

```diff
@@ -390,11 +390,11 @@ async def _libSvcGet(self, name):
         Returns:
             dict: A Storm Service definition.
         '''
-        self.runt.user.confirm(('service', 'get', name))
         ssvc = self.runt.snap.core.getStormSvc(name)
         if ssvc is None:
             mesg = f'No service with name/iden: {name}'
             raise s_exc.NoSuchName(mesg=mesg)
+        self.runt.user.confirm(('service', 'get', ssvc.iden))
```

Comment:
Did we want to have a fallback try here for the other perm and bitch about it to avoid a compat issue for a bit?

Reply:
Change to preserve backward compatibility and complain to users and server side.